server_installation/topics/operator.adoc

```diff
@@ -0,0 +1,14 @@
+
+[[_operator]]
+== {project_operator}
+
+{project_operator} has been designed to automate {project_name} administration in Kubernetes/OpenShift cluster.
+It allows you to:
+
+* Install Keycloak to a namespace
```

Comment:
@slaskawi If examples of steps, that need to be done to achieve this, are provided later in the docs for each of these subcases, would it be possible to make each of these points a link to the section, providing those steps?

Reply:
I don't think so.... This is more a list of features implemented. Actually, it's the same list as here: https://operatorhub.io/operator/keycloak-operator